### PR TITLE
Add provider for Spotify

### PR DESCRIPTION
**Pull request by ker0x**

### CODE REVIEW

File: src/DependencyInjection/Providers/SpotifyProviderConfigurator.php [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    {
+        return 'kerox/oauth2-spotify';
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getLibraryHomepage(): string
+    {
+        return 'https://github.com/kerox/oauth2-spotify';
```
Comment (weaverryan) [inline src/DependencyInjection/Providers/SpotifyProviderConfigurator.php]:
```suggestion
        return 'https://github.com/ker0x/oauth2-spotify';
```

**Comment by ker0x** [comment]
Hello @weaverryan,

Everything should be good now ;)!

File: src/DependencyInjection/Providers/SpotifyProviderConfigurator.php (hunk 1 above)
Comment (bocharsky-bw) [reply src/DependencyInjection/Providers/SpotifyProviderConfigurator.php]: Great catch!

**Comment by weaverryan** [comment]
Thanks @ker0x!

**Pull request merged**